### PR TITLE
Add pepredicates

### PR DESCRIPTION
**Pull request by hoyc1**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hoyc1** [comment]
@conda-forge/help-python, this recipe is ready for review, thanks.

File: recipes/pepredicates/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - numpy
+    - python >=3.6
+    - seaborn
+
+test:
+  imports:
+    - pepredicates
+
+about:
+  home: https://git.ligo.org/will-farr/pepredicates
```
Comment (xhochy) [inline recipes/pepredicates/meta.yaml]: Is there a public homepage / source for this?

File: recipes/pepredicates/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  doc_url: https://docs.ligo.org/will-farr/pepredicates
+  license: GPL3
```
Comment (xhochy) [inline recipes/pepredicates/meta.yaml]: The attachted License is AGPL3, what is correct?

Comment (hoyc1) [reply recipes/pepredicates/meta.yaml]: @xhochy, sorry this was a typo. See commit 501a9d6

File: recipes/pepredicates/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - seaborn
+
+test:
+  imports:
+    - pepredicates
+
+about:
+  home: https://git.ligo.org/will-farr/pepredicates
+  doc_url: https://docs.ligo.org/will-farr/pepredicates
+  license: AGPL3
```
Comment (xhochy) [inline recipes/pepredicates/meta.yaml]: Use SPDX name for this
```suggestion
  license: AGPL-3.0
```

File: recipes/pepredicates/meta.yaml (hunk 1 above)
Comment (hoyc1) [reply recipes/pepredicates/meta.yaml]: I have added the public doc homepage. See commit 73d9279

File: recipes/pepredicates/meta.yaml (hunk 3 above)
Comment (hoyc1) [reply recipes/pepredicates/meta.yaml]: I have changed this in commit 73d9279

**Comment by hoyc1** [comment]
@xhochy, sorry for the delay in replying to your comments but I think I have addressed them all now. Let me know if there is anything else that I need to change. Thanks!

**Pull request merged**